packages/ariakit/src/collection/__examples__/collection/style.css

```diff
@@ -0,0 +1,22 @@
+.collection {
+  @apply
+    overflow-hidden
+    border-none
+    rounded-lg;
+}
+
+.collection-item {
+  @apply
+    px-6
+    py-3
+    flex
+    items-center
+    text-base
+    bg-canvas-1
+    text-canvas-1
+    hover:bg-canvas-1-hover
+    dark:bg-canvas-1-dark
+    dark:text-canvas-1-dark
+    dark:hover:bg-canvas-1-dark-hover
+    focus-visible:ariakit-outline;
+}
```

Comment:
Just to make it simpler:
```suggestion
```

Reply:
Also done in zhttps://github.com/reakit/reakit/pull/977/commits/4f909150cb671cf1738a9a279e2b5be691898b7f